charts/bandstand-web-service/templates/certificate.yaml

```
@@ -24,4 +24,4 @@ spec:
   privateKey:
     algorithm: "RSA"
     size: 2048
-{{- end -}}
+{{ end -}}
```

Pretty sure you'll need a separator too
```suggestion
---
{{ end -}}
```